python-avd/pyavd/_eos_designs/ip_addressing/utils.py

```diff
@@ -101,6 +120,10 @@ def _loopback_ipv4_offset(self: AvdIpAddressing) -> int:
     def _loopback_ipv6_pool(self: AvdIpAddressing) -> str:
         return self.shared_utils.loopback_ipv6_pool
 
+    @cached_property
+    def _loopback_ipv6_prefix_length(self: AvdIpAddressing) -> str:
+        return self.shared_utils.loopback_ipv6_prefix_length
```

Comment:
Maybe this should just be added to fabric_ip_addressing like the other related things. It would typically be a one-time setting.